_src/pldoc/static/js/ui.js

```diff
@@ -12,14 +12,13 @@ $(document).ready(function() {
 
     // smoothscroll to target links
     function smoothScrollLink(e) {
+        var this = e.target,
```

Comment:
Renaming `this` makes me nervous.  It seems like you could get away with `$(e.target).attr('href');` rather.